.github/workflows/actions/deploy-environment/action.yml

```diff
@@ -0,0 +1,128 @@
+name: Deploy to Azure
+
+inputs:
+  environment_name:
+    description: "The name of the environment"
+    required: true
+  image_tag:
+    description: "github SHA of the image"
+    required: true
+  azure_credentials:
+    description: "JSON object containing a service principal that can read from Azure Key Vault"
+    required: true
+  terraform_vars:
```

Comment:
terraform_vars can be determined with the environment name. can we remove the input?